.github/workflows/hermetic_library_generation.yaml

```diff
@@ -24,7 +24,7 @@ env:
 jobs:
   library_generation:
     # skip pull requests coming from a forked repository    
-    if: github.env.REPO_FULL_NAME == github.repository
```

Comment:
@blakeli0 inlining the `org/repo` as an env var seems to break this `if` statement

Reply:
Probably because `REPO_FULL_NAME` is not referenced as a variable like `$REPO_FULL_NAME`?